templates/react/utils/itemToLinks.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {API_PATH} from '../api/_entrypoint';
+
+export default function (itemArray) {
+  if (Array.isArray(itemArray)) {
+    return (
```

Comment:
We can probably even prevent to introduce new files (I would like the generated code to be as minimal as possible) and include this onliner in the list template directly. WDYT @mysiar?